internal/agent/sender/sender_test.go

```diff
@@ -157,3 +121,27 @@ func TestPrintSender(t *testing.T) {
 	got := buf.String()
 	assert.Contains(t, got, want, "Expected %s, got %s", want, got)
 }
+
+func Test_getTargetURL(t *testing.T) {
```

Comment:
Please clarify **all** names of functions. They should be named with Go style

Reply:
done